test/serializers/associations_test.rb

```diff
@@ -290,6 +289,61 @@ def test_illegal_conditional_associations
           assert_match(/:if should be a Symbol, String or Proc/, exception.message)
         end
       end
+
+      class InheritedSerializerTest < ActiveSupport::TestCase
+        InheritedPostSerializer = Class.new(PostSerializer) do
```

Comment:
I'll release a PR later today that gets rid of them, so people stoop copying the patterns they see.